core/src/main/scala/org/apache/spark/sql/SnappySession.scala

```diff
@@ -76,6 +74,7 @@ class SnappySession(@transient private val sc: SparkContext,
 
   self =>
 
+  SnappyContext.initConf = sc.conf
```

Comment:
SnappyStreamingContext has a "val snappySession" field. Lower down in this code there is a "SnappyContext.initGlobalSnappyContext" call in constructor so that should have taken care of initialization of conf no?